Update DB.

GabrielVnM08 opened the pull request:
Dupla: Gabriel Vidal e Pecris Andrade.

GabrielVnM08 committed:
atualizando meu banco com base no principal

GabrielVnM08 committed:
buscando atual repo do principal(by furao)

GabrielVnM08 committed:
criação da tabela logs_erro

GabrielVnM08 committed:
Update tabela logs_erro.

GabrielVnM08 committed:
Update DB.

Adição da tabela dispositivo, sensor e usuário.
Incremento de novas  funcionalidades.
Dupla: Gabriel Vidal e Pecris Andrade.

davifurao commented:
Vai ser retirada a tabela usuarios pois o usuario já é criado pela aplicação laravel

davisouzaluna committed:
Merge branch 'main' into main

davisouzaluna committed:
Update Energy_Monitor.sql

A tabela de usuários irá referenciar para a tabela de dispositivos e a tabela de dispositivos irá  referenciar a tabela sensor
